frontend/packages/text-editor/src/TextEditor.test.tsx

```diff
@@ -204,7 +204,7 @@ describe('TextEditor', () => {
       await expect(onTextIdChange).toHaveBeenCalledWith({ oldId: nb[0].id });
     };
 
-    const getInputs = (name: RegExp) => screen.getAllByRole('textbox', { name });
+    const getInput = (name: RegExp) => screen.getByRole('textbox', { name });
```

Comment:
Should we simplify this code by removing the getInput method/abstraction and use testing-library directly instead?
```suggestion
```